src/main/java/gregtech/loaders/recipe/chemistry/TitaniumRecipes.java

```diff
@@ -0,0 +1,157 @@
+                .fluidOutputs(Ammonia.getFluid(2000))
+                .fluidOutputs(Water.getFluid(1000))
+                .duration(200).EUt(VA[MV]).buildAndRegister();
+    }
+
+    // Advanced separation process for Bauxite
+    private static void bauxiteProcess() {
+        // Bauxite (crushed) + Soda Ash + Calcium Chloride -> Bauxite Slurry
+        MIXER_RECIPES.recipeBuilder()
+                .input(crushed, Bauxite, 32)
+                .input(dust, SodaAsh, 12)
+                .input(dust, CalciumChloride, 6)
+                .fluidInputs(Water.getFluid(4000))
+                .fluidOutputs(BauxiteSlurry.getFluid(4000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite (washed) + Soda Ash + Calcium Chloride -> Bauxite Slurry
+        MIXER_RECIPES.recipeBuilder()
+                .input(crushedPurified, Bauxite, 32)
+                .input(dust, SodaAsh, 12)
+                .input(dust, CalciumChloride, 6)
+                .fluidInputs(Water.getFluid(4000))
+                .fluidOutputs(BauxiteSlurry.getFluid(4000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite Slurry -> Cracked Bauxite Slurry
+        CRACKING_RECIPES.recipeBuilder()
+                .circuitMeta(1)
+                .fluidInputs(BauxiteSlurry.getFluid(16000))
+                .fluidInputs(Steam.getFluid(1000))
+                .fluidOutputs(CrackedBauxiteSlurry.getFluid(16000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite Slurry + Sulfuric -> Aluminium, Slag, Sludge, and SO3 (for looping back to Sulfuric Acid)
+        LARGE_CHEMICAL_RECIPES.recipeBuilder()
+                .fluidInputs(CrackedBauxiteSlurry.getFluid(4000))
+                .fluidInputs(SulfuricAcid.getFluid(1000))
+                .output(dust, Aluminium, 24)
+                .output(dust, BauxiteSlag, 8)
+                .fluidOutputs(BauxiteSludge.getFluid(2500))
+                .fluidOutputs(SulfurTrioxide.getFluid(1000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite Slag -> Salt (looped) + Nd + Cr (byproducts)
+        ELECTROMAGNETIC_SEPARATOR_RECIPES.recipeBuilder()
+                .input(dust, BauxiteSlag)
+                .output(dust, Salt)
+                .chancedOutput(dust, Neodymium, 2000, 250)
+                .chancedOutput(dust, Chrome, 1000, 250)
+                .duration(50).EUt(VA[MV]).buildAndRegister();
+
+        // Bauxite Sludge -> Calcite (looped) + Decalcified Bauxite Sludge
+        DISTILLERY_RECIPES.recipeBuilder()
+                .circuitMeta(1)
+                .fluidInputs(BauxiteSludge.getFluid(500))
+                .output(dust, Calcite, 2)
+                .fluidOutputs(DecalcifiedBauxiteSludge.getFluid(500))
+                .duration(100).EUt(VA[MV]).buildAndRegister();
+
+        // Decalcified Bauxite Sludge -> Rutile, Gallium, SiO2, Iron, Water
+        CENTRIFUGE_RECIPES.recipeBuilder()
```

Comment:
Should this be multiplied up to input amounts of `1000`?

Reply:
Same as below